Workbooks/CosmosDb Fleet/Overview.workbook

```diff
@@ -0,0 +1,2315 @@
+                },
+                {
+                  "durationMs": 86400000,
+                  "createdTime": "2018-08-06T23:52:38.871Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 172800000,
+                  "createdTime": "2018-08-06T23:52:38.871Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 259200000,
+                  "createdTime": "2018-08-06T23:52:38.871Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 604800000,
+                  "createdTime": "2018-08-06T23:52:38.871Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                }
+              ],
+              "allowCustom": true
+            }
+          },
+          {
+            "id": "9af7a941-b4e0-4941-8b28-9972ce6f2bf7",
+            "version": "KqlParameterItem/1.0",
+            "name": "Fleet",
+            "type": 5,
+            "description": "Default subscriptions with CosmosDB accounts. If you are not seeing a subscription, make sure to add it to your default subscriptions. ",
+            "isRequired": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.documentdb/fleets": true
+              },
+              "additionalResourceOptions": [
+                "value::1"
+              ],
+              "showDefault": false
+            }
+          },
+          {
+            "id": "57754aaa-671b-412e-84fe-b1258caacf3a",
+            "version": "KqlParameterItem/1.0",
+            "name": "Fleetspaces",
+            "type": 2,
+            "multiSelect": true,
+            "quote": "",
+            "delimiter": ",",
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Fleet}/fleetspaces\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2024-12-01-preview\"},{\"key\":\"\",\"value\":\"\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value.*.name\",\"columns\":[]}}]}",
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "showDefault": false
+            },
+            "queryType": 12,
+            "value": [
+              "fleetspace1"
+            ]
+          },
+          {
+            "id": "18f1c50e-63d0-47f4-ab76-b2ebc09bbe49",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resources",
+            "label": "Database Accounts",
```

Comment:
Should this be labelled Fleetspace Accounts?

Reply:
FleetspaceAccounts are the same as database accounts. They are a reference to databaseaccounts, in the portal we call the new blade to list accounts as "DatabaseAccounts" so keeping that